### ISSUE TITLE
S2-MLP: Spatial-Shift MLP Architecture for Vision

### ISSUE THREAD

**Issue by rentainhe** [rentainhe/what_I_have_read#1]
## Papers
[S2-MLP: Spatial-Shift MLP Architecture for Vision](https://arxiv.org/abs/2106.07477)

## Spatial Shift MLP Block
![image](https://user-images.githubusercontent.com/48727989/127729249-9dd34b14-228a-4875-9a16-3dce3f8e14fc.png)

## Contribution
- Remove `token-mixing` 操作，降低过拟合风险，only `channel-mixing`
- 引入local特性，设计spatial shift来进行空间移位操作，对移位后的特征进行channel维度的MLP，来捕捉不同分组之间不同局部空间spatial特征的交互

## Training Hyper-parameters
- training strategy followed [DeiT](https://arxiv.org/abs/2012.12877)

Additional Details:
- **AdamW** with weight decay 0.05
- Batch size 1024
- Linear warm-up and cosine decay
- Initial learning rate 1e-3 and gradually drops to 1e-5
- Epoch 300
- Label smoothing
- [DropPath](https://arxiv.org/abs/1605.07648)
- [Repeated augmentation](https://openaccess.thecvf.com/content_CVPR_2020/papers/Hoffer_Augment_Your_Batch_Improving_Generalization_Through_Instance_Repetition_CVPR_2020_paper.pdf)
